FBAllocationTracker/NSObject+FBAllocationTracker.h

```diff
@@ -31,3 +31,12 @@
 @end
 
 #endif // _INTERNAL_FBAT_ENABLED
+
+/// Returns whether `object` is the zombie object
+///
+/// Returns `true` if `object` is nil
```

Comment:
Hey, thanks! Let's be consistent about comments though. We are using block comments in form:
```
/**
 Comment
*/
```